packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/process/dq-response.ts

```diff
@@ -87,13 +88,27 @@ function parseDocumentReference(
     return title;
   };
 
+  const sizeValue = findSlotValue("size");
+  const repositoryUniqueId = findSlotValue("repositoryUniqueId");
+  const docUniqueId = findExternalIdentifierValue(XDSDocumentEntryUniqueId);
+
+  if (!repositoryUniqueId || !docUniqueId) {
+    const msg = "Document Reference is missing repositoryUniqueId or docUniqueId";
```

Comment:
capture but dont throw an error and potentially halt process of processing other document references